paradox-menu.el

```diff
@@ -727,6 +728,7 @@ Status:  _i_nstalled _a_vailable _d_ependency _b_uilt-in
      ["By Package Name" paradox-sort-by-package]
      ["By Status (default)" paradox-sort-by-status]
      ["By Number of Stars" paradox-sort-by-★]
+     ["By Number of Downloads" paradox-sort-by-↓]
```

Comment:
I matched the previous behavior, but probably better to make use of `paradox--column-name-star` and `paradox--column-name-download` rather than hardcode.